src/main/webapp/app/exam/participate/exam-participation.component.ts

```diff
@@ -671,6 +674,19 @@ export class ExamParticipationComponent implements OnInit, OnDestroy, ComponentC
         }
     }
 
+    /**
+     * Resets the pageComponentVisited array by setting all elements to false, and then sets the element
+     * at the specified activePageIndex to true, if provided and within the array bounds.
+     *
+     * @param {number} activePageIndex - The index of the currently active exercise page in the pageComponentVisited array.
+     */
+    private resetPageComponentVisited(activePageIndex: number) {
+        this.pageComponentVisited = this.pageComponentVisited.fill(false);
```

Comment:
```suggestion
        this.pageComponentVisited.fill(false);
```

`.fill` modifies the array, so you could leave out the left half.

Reply:
True, I was unsure about that since the method description was not that clear, but it works the same.